.github/workflows/run-kata-coco-tests.yaml

```diff
@@ -99,7 +110,7 @@ jobs:
         run: bash tests/integration/kubernetes/gha-run.sh cleanup-snapshotter
 
       - name: Delete CoCo KBS
-        if: always()
+        if: ${{ matrix.machine != 'tdx-no-attestation' }}
```

Comment:
Do we need something like `if: ${{ always() && matrix.machine != 'tdx-no-attestation' }}` here to ensure that it runs even if there are failures in previous steps?

Reply:
That's a very good point, I will address the comment.